spec/models/trust_data_file_spec.rb

```diff
@@ -0,0 +1,112 @@
+require 'rails_helper'
+
+RSpec.describe TrustDataFile, type: :model do
+  describe '#trusts' do
+    let(:filename) { Rails.root.join('tmp/trust_test_data.csv') }
+
+    context 'when a trust is a single-academy or multi-academy and has a companies house number' do
+      let(:attrs) do
+        [
+          {
+            group_uid: '1001',
+            companies_house_number: '01234222',
+            name: 'Little Hampton Academy',
+            address_1: '12 High St',
+            address_2: 'Little Hampton Count',
+            town: 'London',
+            postcode: 'NW1 1AA',
+            status: 'Open',
+            group_type: 'Multi-academy trust',
+          },
+          {
+            group_uid: '4001',
+            companies_house_number: '09933123',
+            name: 'Callio Forsythe Academy',
+            address_1: 'Big Academy',
+            address_2: 'Strange Lane',
+            town: 'Easttown',
+            postcode: 'EW1 1AA',
+            status: 'Closed',
+            group_type: 'Single-academy trust',
+          },
+        ]
+      end
+
+      before do
+        create_trust_csv_file(filename, attrs)
+      end
+
+      after do
+        remove_file(filename)
+      end
+
+      it 'retrieves the trust data' do
+        trusts = TrustDataFile.new(filename).trusts
+        expect(trusts.first).to include(
+          gias_group_uid: '1001',
+          companies_house_number: '01234222',
+          name: 'Little Hampton Academy',
+          address_1: '12 High St',
+          address_2: 'Little Hampton Count',
+          town: 'London',
+          postcode: 'NW1 1AA',
+          status: 'open',
+          organisation_type: 'Multi-academy trust',
+        )
+        expect(trusts.second).to include(
+          gias_group_uid: '4001',
+          companies_house_number: '09933123',
+          name: 'Callio Forsythe Academy',
+          address_1: 'Big Academy',
+          address_2: 'Strange Lane',
+          town: 'Easttown',
+          postcode: 'EW1 1AA',
+          status: 'closed',
+          organisation_type: 'Single-academy trust',
+        )
+      end
+    end
+
+    context 'when a trust is has no companies house number or is not a single or multi academy trust' do
```

Comment:
```suggestion
    context 'when a trust has no Companies House number or is not a single- or multi-academy trust' do
```

Reply:
Yes, I did think about this and wanted to do just that. I think in the case of the CHN it's used as the unique identifier when looking them up and there were a handful of Trusts that had blank numbers (all closed trusts) but we're now pulling in closed ones and I thought that might just complicate things to have blank ones. If they were ever populated in GIAS we'd pull them in so it seemed fine?  We could pull in the other entities from the file if it was useful though if that's what you meant (i.e. ignore the SAT/MAT filter)?

Reply:
> We could pull in the other entities from the file if it was useful though if that's what you meant (i.e. ignore the SAT/MAT filter)?

I think it's fine the way it is, I was just thinking out loud, it's easy enough to change later and re-fill if these assumptions aren't right...